arlo_server/__init__.py

```diff
@@ -1293,11 +1319,20 @@ def set_loggedin_user(user_type: UserType, user_email: str):
     session["_user"] = {"type": user_type, "email": user_email}
 
 
-def get_loggedin_user():
+def get_loggedin_user() -> Union[Tuple[UserType, str], Tuple[None, None]]:
     user = session.get("_user", None)
     return (user["type"], user["email"]) if user else (None, None)
 
 
+def get_loggedin_user_record():
```

Comment:
The user's type is purely a session concern, so we can't just return the user record. I had both functions because sometimes we didn't actually need the `User` record and I just left those uses as-is. I'm up for alternatives.

Reply:
"The user's type is purely a session concern" kind of gives me worries. I'm not totally sure why yet. 

I guess what I'm inferring from that statement (and from a test case I saw) is that users can log in as different types. Do we have a use case for that feature? And do we have a way to restrict what types a user can log in as?

I'm also thinking about situations down the road where we might want to query users by their type, and not sure how we'd go about that.

Re: the two functions, I think it would be simpler for me to think about if they were just consolidated. The only current case I see that uses the one without the `User` record is logout, and I don't think it's a big deal to hit the db once there.

Reply:
There are two types right now described by the `UserType` enum: audit admins and jurisdiction admins. There will be more (such as audit board members), but for now those are the only two we create a session for. There are two hardcoded entry points for each of these types, so we don't currently have to worry about someone logging in as something they shouldn't.

As for querying based on it, that's not really thinking of it the right way. Firstly, there will be effectively zero users who are both, but the database does allow it. It is a login session where it matters what the type is because you're logging into a particular view of the data. You explicitly choose which one when you log in, much like e.g. a 401k employee vs a 401k administrator.

Reply:
Sounds good, my vote is for consolidating to one function, but this is ok too